tests/unit/serializers/rest-serializer-test.js

```diff
@@ -1,10 +1,17 @@
 import RestSerializer from 'ember-cli-mirage/serializers/rest-serializer';
-
+import { Model } from 'ember-cli-mirage';
+import Schema from 'ember-cli-mirage/orm/schema';
+import Db from 'ember-cli-mirage/db';
 import {module, test} from 'qunit';
 
 module('Unit | Serializers | RestSerializer', function(hooks) {
   hooks.beforeEach(function() {
-    this.serializer = new RestSerializer();
+    let schema = new Schema(new Db(), {
+      person: Model
+    });
+    this.serializer = new RestSerializer({
+      schema
+    });
```

Comment:
Were these needed/used for anything?

Reply:
Yes. Without them this test blows up because there is no `schema` attached and then for various other reasons when any of these parts is removed.